### ISSUE TITLE
multiphase_flow/ex2 is too long

### ISSUE THREAD

**Issue by drwells** [IBAMR/IBAMR#288]
I let this run for a few days. Even though the final time for the test is 10 (not unreasonable), the test has only gotten to about `t = 0.8` over the last three days (i.e., this test would need about a month to finish). We should shorten this by as much as possible.

This is also an issue with the example input file `input2d.cyl` (its identical to `input2d.test`).

**Comment by amneetb** [comment]
For multiphase_flows/ex1 and multiphase_flows/ex2 the end time should be around 0.7 in input2d.cyl/input2d.test files. The cylinder reaches the bottom domain by then. Any more runtime forces it to impinge on the wall with prescribed velocity, then leads to reduction of time step size.

<img width="562" alt="screen shot 2018-08-15 at 11 37 04 am" src="https://user-images.githubusercontent.com/7378152/44166232-b30c0400-a07f-11e8-8a1d-a2ad9274e7fe.png">

**Comment by drwells** [comment]
It looks like the time step stays at 5e-05 until about t = 0.74, at which point it starts decreasing a lot.

We still need 14000 time steps to get that far.

**Comment by amneetb** [comment]
I am wondering if we should split tests into two categories: one that require eyeballing or steady state solutions (long runs) and one that are quick to run and do quantitative number checking? This test is supposed to be long to check for interface evolution and stability.

**Comment by drwells** [comment]
I think that's reasonable. In principle we do have `gtest-long` but the tests are really not that different from `gtest` (and this one, in particular, is in `gtest`).

**Comment by boyceg** [comment]
I think that it is worth emphasizing that the gtest tests are software tests, which are different from method verification/validation tests (although they can also be method verification/validation tests if those tests are relatively fast, or they could be short sections of those kinds of tests). The purpose of the gtest tests is to identify regressions in the implementation.

Imagine that someone accidentally did something to reduce the order of a fluid solver. (Who would have ever done such a terrible thing? Well, I did once, and it took me a while to notice it, because I did not have automated regression tests!) One would not need to run for 1000’s of time steps to notice this regression; there would be a change in the output after even a small number of time steps.

In most cases, I think that for the gtests, we just want to run for long enough for something nontrivial to happen, and then compute some statistics about the result that are easy to check.
